.github/actions/setup-sql-server/Initialize-SqlServer.ps1

```diff
@@ -1,10 +1,14 @@
-﻿if ($Env:RUNNER_OS -eq "Windows")
+if ($Env:RUNNER_OS -eq "Windows")
 {
     choco install sql-server-express --no-progress
 }
 else
 {
     docker pull mcr.microsoft.com/mssql/server &&
-    docker run --name sql2019 -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=Password1!' -p 1433:1433 -d 'mcr.microsoft.com/mssql/server:2019-latest' &&
-    docker exec -u 0 sql2019 bash -c 'mkdir /data; chmod 777 /data -R; chown mssql:root /data'
+    docker run --name sql2019 `
+        --env 'ACCEPT_EULA=Y' `
+        --env 'SA_PASSWORD=Password1!' `
+        --publish 1433:1433 `
+        --detach 'mcr.microsoft.com/mssql/server:2019-latest' &&
```

Comment:
If you want to put each argument on a new line, let's start with the first one:
```suggestion
    docker run `
        --name sql2019 `
        --env 'ACCEPT_EULA=Y' `
        --env 'SA_PASSWORD=Password1!' `
        --publish 1433:1433 `
        --detach 'mcr.microsoft.com/mssql/server:2019-latest' &&
```

Reply:
But then again I thought that we'd switch to using maps for parameters wherever necessary to avoid the barely visible backticks, like e.g. in [`build-dotnet`](https://github.com/Lombiq/GitHub-Actions/blob/dev/.github/actions/build-dotnet/action.yml#L128).

Reply:
I'd argue that if they're barely visible then one needs to increase the font size :D. Jokes aside, in the linked case I can understand, because it's YML mixed with PS mixed with GitHub expressions. This here is a super plain command invocation though.

Reply:
I'll take your word for it. Let's just hope that no-one will waste any time on debugging a missing `` ` `` here.

Reply:
I think the problem isn't that you don't see the backtick but that you don't see if there is a space or other non-printing character between it and the newline. Also you can never know if some parsers crush the code into a single line before interpreting anyway, then what the backtick escapes is completely implementation-dependent. These can lead to some debugging horror which is why I avoid escaping newlines in every language.